lib/vmdb/fast_gettext_helper.rb

```diff
@@ -1,4 +1,5 @@
 require 'vmdb/gettext/domains'
+require 'vmdb/plugins'
```

Comment:
I had thought autoload would take care of this...surprised it's needed.

Reply:
Certainly seems so in [Vmdb::Settings.template_roots](https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/settings.rb#L158).  Maybe something else is requiring it earlier?

Reply:
On the other hand, [Vmdb::Inflections](https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/inflections.rb#L54) *does* require it.

Reply:
if it is more intuitive, can make it match what inflections does by changing it to require_relative and putting it inside the method.